docs/versioned_docs/version-v1.10/guides/kubernetes-helm-chart.mdx

```diff
@@ -163,7 +163,7 @@ the ORY Hydra Helm Chart
 
 ```bash
 $ helm install hydra-example ory/hydra \
-    --set 'hydra.config.secrets.system=$(LC_ALL=C tr -dc 'A-Za-z0-9' < /dev/urandom | base64 | head -c 32)' \
+    --set 'hydra.config.secrets.system={'$(LC_ALL=C tr -dc 'A-Za-z0-9' < /dev/urandom | base64 | head -c 32)'}' \
```

Comment:
Shouldn’t this be []? Could you maybe point to docs which explain this? :)

Reply:
Thanks a lot @aeneasr for your review and help. I have tested this with your suggestion [] and it errors out as earlier `"Expected hydra.config.secrets.system to be a list of strings"` .  So, the reason I added {} around the command is because Lists can be expressed by enclosing values in { and } when passing using `--set` flag in helm.  For example, --set name={a, b, c} translates to:

```
name:
  - a
  - b
  - c
```

Below is helm doc[1] I referred, suggested and tested out the changes in my local setup

[1] https://helm.sh/docs/intro/using_helm/

Reply:
Awesome, thank you!